test/built-ins/TypedArray/prototype/fill/fill-values-conversion-operations-consistent-nan.js

```diff
@@ -73,7 +73,12 @@ features: [TypedArray]
 ---*/
 
 testWithTypedArrayConstructors(function(FA) {
-  var precision = FA === Float32Array ? "single" : "double";
+  var precision =
+    FA === Float32Array
+    ? "single"
+    : (typeof Float16Array !== "undefined" && TA === Float16Array)
```

Comment:
```suggestion
    : FA === Float16Array
```
TA → FA, but also when could this ever be `Float16Array` when the global isn't present?

Reply:
We want to avoid
```
Uncaught ReferenceError: Float16Array is not defined
```

Reply:
ahh duh, not sure how i missed that (－‸ლ)

Reply:
```suggestion
    : (typeof Float16Array !== "undefined" && FA === Float16Array)
```